profiles/common/modules/custom/nexteuropa_json_field/nexteuropa_json_field.module

```diff
@@ -39,8 +38,21 @@ function nexteuropa_json_field_field_instance_settings_form($field, $instance) {
       '#default_value' => isset($settings['js_to_load']) ? $settings['js_to_load'] : array(),
       '#options' => $options,
       '#description' => t('Select external javascripts to load for this field'),
+      '#weight' => 1,
+    );
+
+    $form['defer_on_js_to_load'] = array(
+      '#type' => 'checkboxes',
+      '#title' => t('Add the "refer" attribute to'),
```

Comment:
Should be "defer", no ?

Reply:
Oops, it will be fixed via the merge from 2.4 to 2.5